components/role-mgt/org.wso2.carbon.identity.role.v2.mgt.core/pom.xml

```
@@ -0,0 +1,173 @@
+        <dependency>
+            <groupId>org.wso2.carbon.identity.framework</groupId>
+            <artifactId>org.wso2.carbon.identity.api.resource.mgt</artifactId>
+        </dependency>
+    </dependencies>
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.felix</groupId>
+                <artifactId>maven-bundle-plugin</artifactId>
+                <extensions>true</extensions>
+                <configuration>
+                    <instructions>
+                        <Bundle-SymbolicName>
+                            ${project.artifactId}
+                        </Bundle-SymbolicName>
+                        <Bundle-Name>${project.artifactId}</Bundle-Name>
+                        <Private-Package>
+                            org.wso2.carbon.identity.role.v2.mgt.core.internal
+                        </Private-Package>
+                        <Import-Package>
+                            org.apache.commons.lang; version="${commons-lang.wso2.osgi.version.range}",
+                            org.apache.commons.logging; version="${import.package.version.commons.logging}",
+                            org.apache.commons.collections; version="${commons-collections.wso2.osgi.version.range}",
+                            org.apache.axiom.*; version="${axiom.osgi.version.range}",
+                            org.osgi.framework; version="${osgi.framework.imp.pkg.version.range}",
+                            org.osgi.service.component; version="${osgi.service.component.imp.pkg.version.range}",
+                            org.wso2.carbon; version="${carbon.kernel.package.import.version.range}",
+                            org.wso2.carbon.context; version="${carbon.kernel.package.import.version.range}",
+                            org.wso2.carbon.user.core.*; version="${carbon.kernel.package.import.version.range}",
+                            org.wso2.carbon.user.api; version="${carbon.user.api.imp.pkg.version.range}",
+                            org.wso2.carbon.user.mgt.*; version="${carbon.identity.package.import.version.range}",
+                            org.wso2.carbon.utils.*; version="${carbon.kernel.package.import.version.range}",
+                            org.wso2.carbon.database.utils.*;version="${org.wso2.carbon.database.utils.version.range}",
+                            org.wso2.carbon.identity.core.util;version="${carbon.identity.package.import.version.range}",
+                            org.wso2.carbon.identity.event.*; version="${carbon.identity.package.import.version.range}",
+                            org.wso2.carbon.identity.application.common.*;version="${carbon.identity.package.import.version.range}"
+                            org.wso2.carbon.identity.application.mgt.*; version="${carbon.identity.package.import.version.range}",
+                            org.wso2.carbon.identity.api.resource.mgt.*; version="${carbon.identity.package.import.version.range}",
```

```suggestion
                            org.wso2.carbon.identity.application.mgt.*; version="${carbon.identity.package.import.version.range}",
                            org.wso2.carbon.identity.api.resource.mgt.*; version="${carbon.identity.package.import.version.range}",
                            org.wso2.carbon.identity.application.common.*;version="${carbon.identity.package.import.version.range}"
```
change the order to resolve activation issue